packages/deploy-helpers/deploy-logger.js

```diff
@@ -2,28 +2,56 @@ const flatten = require('truffle-flattener')
 const mkdirp = require('mkdirp')
 const fs = require('fs')
 const path = require('path')
-
-const truffleConfig = require('@aragon/truffle-config-v4/truffle-config')
+const findUp = require('find-up')
 
 const FLATTEN_DIR = './flattened_contracts'
 
-module.exports = async (instance, { verbose = true, flattenContracts = true } = {}) => {
+const getTruffleConfig = () => {
+  try {
+    let truffleConfigPath = findUp.sync('truffle.js')
```

Comment:
To be strictly compatible with truffle, maybe we should look for `truffle-config.js` before `truffle.js` since it is their behavior: https://github.com/trufflesuite/truffle/blob/develop/packages/config/src/index.ts#L156

Reply:
Indeed!